src/common/Types.ts

```diff
@@ -338,6 +338,10 @@ export interface CommonAccessibilityProps {
 
     // Desktop only.
     tabIndex?: number;
+
+    // iOS only.
+    accessibilityActions?: string[];
+    onAccessibilityAction?: Function;
```

Comment:
This function should have a well-defined type (including incoming and return params) rather than a generic Function type.